File: Backend/src/main/java/com/ms/invcms/company/Company.java
package com.ms.invcms.company;

import com.ms.invcms.base.BaseEntity;
import com.ms.invcms.phone.Phone;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@EqualsAndHashCode(callSuper = true)
@Data
@Entity
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "company")
public class Company extends BaseEntity {

    private String name;
    private String logo;
    private String taxIdentificationNumber;
    private String taxOffice;

    @OneToOne
    private Phone phone;
    private String address;
    private String description;



}

File: Frontend/invcmsWeb/src/app/domain/company.ts
import { Phone } from './phone'


export class Company{
      id:string
      name : string;
      taxIdentificationNumber:string;
      taxOffice:string;
      phone:Phone;
      address:string;
      description:string;
      logo:string

      constructor() {
        this.id=null
        this.name = null
        this.taxIdentificationNumber = null
        this.taxOffice = null
        this.phone = null
        this.address = null
        this.description=null
        this.logo=null
    }

    
}